src/language-js/print/array.js

```diff
@@ -125,4 +136,32 @@ function printArrayItems(path, options, printPath, print) {
   return printedElements;
 }
 
+function printArrayItemsUsingFill(path, options, printPath, print) {
```

Comment:
nit: what not an arrow function? (im an outsider,
dont think ive contributed to prettier before)

Reply:
Function declarations are more convenient thanks to hoisting.